tools/start_server.py

```diff
@@ -258,6 +281,11 @@ def __init__(self):
 
     self.webengine_apps = {}
 
+    self.file_server = WebengineFileServer(Flags.FILE_SERVER_HOST, Flags.FILE_SERVER_PORT, Flags.FILE_SERVER_REMOTE_HOST, Flags.FILE_SERVER_REMOTE_PORT)
```

Comment:
hmm the instance of the file_server in the WebEngine manager uses the flags but I don't think we should restrict the class itself to only accept one (host, port) mapping. For example, if there is a case in the future where I have to create multiple file servers (perhaps pointing to different directories) it would be easier to create multiple instances this way.

Reply:
I am not proposing we restrict the class at all, just suggesting we use the Flags directly where they are needed. This approach is more verbose, passing the flags here and then creating another variable to hold them versus just using the Flags directly where they are needed.

Why would this way be easier to create multiple instances?

Reply:
in regards to the multiple instances scenario if I make this change to the FileServer
```
 self.tcp_server = socketserver.TCPServer((Flags.FILE_SERVER_HOST, Flags.FILE_SERVER_PORT), self.getRequestHandler(self.app_dir_mapping))
```
create two instances of the file server in WebEngineManager
```
self.file_server =  WebengineFileServer()
self.file_server_2 = WebengineFileServer()
```
and tried to start both one would start the second would not since they’re using the same host and port for all instances, right?
Whereas at the moment I could add extra flags and pass those into the second instance to create two file servers

Reply:
Ah I see where you're coming from now. That's a fair point.